.travis.yml

```diff
@@ -162,6 +162,20 @@ matrix:
         - mbedtools compile -t GCC_ARM -m K64F
         - ccache -s
 
+    - <<: *mbed-tools-test
+      name: "Test custom target"
+      env: NAME=test-import-cmd CACHE_NAME=test-import-cmd
+      script:
+        - mbedtools new custom-target-test
+        - cd custom-target-test
+        - cp $TRAVIS_BUILD_DIR/travis-ci/test-data/custom_targets.json .
+        - mkdir TARGET_IMAGINARYBOARD
+        - cp $TRAVIS_BUILD_DIR/travis-ci/test-data/TARGET_IMAGINARYBOARD/* TARGET_IMAGINARYBOARD
+        - cp mbed-os/targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_DISCO_L475VG_IOT01A/P* TARGET_IMAGINARYBOARD
+        - sed -i '/add_subdirectory(${MBED_PATH})/i add_subdirectory(TARGET_IMAGINARYBOARD)' CMakeLists.txt
```

Comment:
Does this replace the line `add_subdirectory(${MBED_PATH})` with `add_subdirectory(TARGET_IMAGINARYBOARD)`? We'd like to add `add_subdirectory(TARGET_IMAGINARYBOARD)` before the  `add_subdirectory(${MBED_PATH})` line.

Reply:
OK, `/i` is the trick here. This looks good.